src/common/utils/create-location/parse-path.ts

```diff
@@ -0,0 +1,13 @@
+export function parsePath(path: string) {
+  const url = new URL(path, 'ws://a.a');
```

Comment:
Out of curiosity, why `ws://a.a` ?

Reply:
Its the shortest url I could come up with 😅